.github/workflows/main.yaml

```diff
@@ -614,7 +626,7 @@ jobs:
         run: |
           shopt -s nullglob
           cd artifacts
-          FILES=$(ls {*.tar.gz,*.txt})
+          FILES=$(ls {*.tar.gz,*.zip,*.txt})
```

Comment:
at the moment the .zip is not yet included in the release, I'm not sure how .exe ends up here. I assume it's via some `actions/download-artifact` but I couldn't figure it out. Any help would be appreciated.